### PR TITLE
WebGLProgram: add /* glsl */ template. 

### PR DESCRIPTION
**Pull request by linbingquan**

### CODE REVIEW

**Comment by Mugen87** [comment]
What is the motivation behind this change?

**Comment by linbingquan** [comment]
> What is the motivation behind this change?

TBH, I just want to highlight the code like `/src/renderers/ShaderLib/**.glsl.js`.

|Before|After|
|:---:|:---:|
|image|image|

**Comment by Mugen87** [comment]
Not sure it makes sense for this part of the code. Besides, it seems the PR breaks examples.

**Comment by linbingquan** [comment]
> Not sure it makes sense for this part of the code. Besides, it seems the PR breaks examples.

Thank you for your reply. I made a mistake. This PR will OK.

File: src/renderers/webgl/WebGLProgram.js [hunk 1]
```diff
@@ -673,27 +673,29 @@ function WebGLProgram( renderer, cacheKey, parameters, bindingStates ) {
...
+			#define attribute in
+			#define varying out
+			#define texture2D texture
+			${prefixVertex}
+		`;
+
+		prefixFragment = /* glsl */`
+			#define varying in
+			${( parameters.glslVersion === GLSL3 ) ? '' : 'out highp vec4 pc_fragColor;'}
+			${( parameters.glslVersion === GLSL3 ) ? '' : '#define gl_FragColor pc_fragColor'}
```
Comment (Mugen87) [inline src/renderers/webgl/WebGLProgram.js:686]: Can you please verify if this statement actually works? If this code breaks, it is not visible with a broken E2E test.

Comment (linbingquan) [reply src/renderers/webgl/WebGLProgram.js:686]: Sorry, I don't how to verify it, you mean `npm run test-e2e`?

Comment (Mugen87) [reply src/renderers/webgl/WebGLProgram.js:686]: You need a special test case for this. Fortunately, I still have a fiddle around: https://jsfiddle.net/rnbqyx25/

The idea is to use a `ShaderMaterial` with GLSL 3.0 code and check if `out highp vec4 pc_fragColor;` and `#define gl_FragColor pc_fragColor` is NOT set.

**Comment by mrdoob** [comment]
The reason it's breaking is because the current rollup code can't handle javascript in inside a `/*glsl*/` string.

**Comment by Mugen87** [comment]
@mrdoob Not 100% sure but I don't think the code breaks because of rollup since the scripts only processes code in `glsl.js` files. The PR is about changing code in `WebGLProgram`.

**Comment by mrdoob** [comment]
Ah, true. Well, I think I just tried doing this myself in the past and that's where I remember things breaking for me.

**Comment by Mugen87** [comment]
Closing until it's proved that `ShaderMaterial` with GLSL 3.0 code does not break with this change.

**Pull request closed without merging**